sbin/common-functions.sh

```diff
@@ -28,7 +28,7 @@ determineBuildProperties() {
 
     BUILD_TYPE=normal
     DEFAULT_BUILD_FULL_NAME=${OS_KERNEL_NAME}-${OS_MACHINE_NAME}-${BUILD_TYPE}-${JVM_VARIANT}-release
-    BUILD_FULL_NAME=${BUILD_FULL_NAME:-"$DEFAULT_BUILD_FULL_NAME"}
+    export BUILD_FULL_NAME=${BUILD_FULL_NAME:-"$DEFAULT_BUILD_FULL_NAME"}
```

Comment:
One of the entry points - will check, fairly sure it does not affect anything, all the builds I ran this morning across all our platforms work fine despite this change, it takes the existing value and updates the env variable